form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -320,6 +321,13 @@ internal fun FieldProvider(
                     resources = resources,
                 )
             }
+            ValueType.REFERENCE,
+            ValueType.GEOJSON,
+            ValueType.USERNAME,
+            ValueType.TRACKER_ASSOCIATE,
+            -> {
+                InputNotSupported(title = fieldUiModel.label)
+            }
 
             else -> {
```

Comment:
why do we need to maintain the else which are the remaining valuetypes?

Reply:
I think you are right, we don't need it anymore